src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.html

```diff
@@ -314,9 +318,12 @@ <h4>
                     </table>
 
                     <ng-template #noComplaints>
-                        <div class="alert alert-info">
+                        <div *ngIf="numberOfTutorComplaints === 0" class="alert alert-info">
                             {{ 'artemisApp.tutorExerciseDashboard.noComplaints' | translate }}
                         </div>
+                        <div *ngIf="numberOfTutorComplaints != 0" class="alert alert-info">
```

Comment:
```suggestion
                        <div *ngIf="numberOfTutorComplaints !== 0" class="alert alert-info">
```

Reply:
not sure if this would be more correct, but this would be more consistent to the `===` above